Assets/__Scripts/MapEditor/UI/Placement Controller UI/ColorTypeController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ColorTypeController : MonoBehaviour
+{
+    [SerializeField] private NotePlacement notePlacement;
+    [SerializeField] private Image leftSelected;
+    [SerializeField] private Image rightSelected;
+    [SerializeField] private Image leftNote;
+    [SerializeField] private Image leftLight;
+    [SerializeField] private Image rightNote;
+    [SerializeField] private Image rightLight;
+
+    void Start()
+    {
+        leftSelected.enabled = true;
+        rightSelected.enabled = false;
+        StartCoroutine(SetupColors());
+    }
+
+    private IEnumerator SetupColors()//PlatformDescriptor is in another scene so we have to find it, can probably change this get the colors from somewhere else in the future
```

Comment:
the static `LoadInitialMap.PlatformLoadedEvent` does exist, so this enumerator can pretty much be replaced by a subscription to that event.

it's a bit weird, especially considering the class it resides in, but this will all be remedied with the zenject implementation